src/App.tsx

```diff
@@ -15,71 +16,65 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+export const App: React.FC = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {selectedGood ? `${selectedGood} is selected` : 'No goods selected'}
+        {selectedGood && (
+          <button
+            data-cy="ClearButton"
+            title="clearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={() => setSelectedGood('')}
```

Comment:
```suggestion
            onClick={resetGood}
```
Consider moving this logic from jsx